### PR TITLE
Add a script to pre-generate changelog entries

### PR DESCRIPTION
**Pull request by astorije**
I have been using and improving that script for some time now, at least all 2.5 and 2.6 releases. 👌

### CODE REVIEW

File: scripts/.eslintrc.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+
+# Necessary to support async/await... grumble grumble...
+parserOptions:
+  ecmaVersion: 8
+
+rules:
+  space-before-function-paren:
```
Comment (xPaw) [inline scripts/.eslintrc.yml]: Can you modify eslintrc in root to match this?

File: scripts/.eslintrc.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+
+# Necessary to support async/await... grumble grumble...
+parserOptions:
+  ecmaVersion: 8
```
Comment (xPaw) [inline scripts/.eslintrc.yml]: You can do this with a eslint comment within the `changelog.js` file instead of making a config file.

Comment (astorije) [reply scripts/.eslintrc.yml]: It seems to me that `parserOptions` configs cannot be part of an ESLint comment 💩
Or at least I couldn't find how... Any idea?

File: scripts/.eslintrc.yml (hunk 1 above)
Comment (astorije) [reply scripts/.eslintrc.yml]: Sure thing! It's kinda clumsy though in the current shape, do you mind if I change overall into `space-before-function-paren: error` (i.e. `always`)? It's seems way more common to me and ESLint's default.
Plus I can do that automatically with `--fix` to not break anything.

Will do in another PR if you're okay with this.

Comment (xPaw) [reply scripts/.eslintrc.yml]: Oh, that's gonna affect so much that I wouldn't recommend doing it.

Comment (astorije) [reply scripts/.eslintrc.yml]: Yeah, about 400 `function()`... I really hate those, but whatever.

Anyway, moved this to the root config!

**Comment by astorije** [comment]
Merging this now as I'm going to use it to draft the v2.6.0 release. See you on the other side!

**Pull request merged**